db/scripts/cleanup_scheduled_calls.js

```diff
@@ -14,19 +17,44 @@ async function getDb() {
 
 async function cleanupScheduledCalls() {
   const db = await getDb();
```

Comment:
Now that this file is running as part of the test suite, we should close the connection once we're done so that there are no loose connections hanging around (like this:  https://github.com/madetech/nhs-virtual-visit/blob/df12a1ff523480b840aca16f282ccfba7143d2b4/testTeardown.js#L14-L17

We may need to pass the db in the result so that we can close it in a ["finally"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally) call at the end

Reply:
I'd already done this, but in the script itself rather than the test teardown. I think it should be there because if the script is run manually it waits until it times out rather than completing immediately otherwise.